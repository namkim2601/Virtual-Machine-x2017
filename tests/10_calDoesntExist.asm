FUNC LABEL 0
    CAL VAL 7
    RET
