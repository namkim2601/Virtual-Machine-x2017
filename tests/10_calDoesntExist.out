Error: Trying to call function that does not exist
